lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -692,11 +692,27 @@ def _wait_for_elb_interface_removed(self):
 
     @_throttleable_operation(_THROTTLING_RETRIES)
     def _get_elb(self):
-        elbs = self.elb_conn.get_all_load_balancers()
-        for elb in elbs:
-            if self.name == elb.name:
-                self.status = 'ok'
-                return elb
+        token = None
+        while True:
+            elbs = self.elb_conn.get_all_load_balancers(marker=token)
```

Comment:
But this module accepts a list of ELB names, or none at all (meaning return info on all ELBs), so we can't assume that we are only looking up a single ELB.